reactor-netty-http/src/main/java/reactor/netty/http/server/logging/AccessLogArgProvider.java

```diff
@@ -28,12 +29,12 @@
 public interface AccessLogArgProvider {
 
 	/**
-	 * Returns the date-time string with a time-zone, (e.g. "30/Oct/2020:03:52:11 +0000").
+	 * Returns the date-time of the moment when the request was received
 	 *
-	 * @return the date-time string with a time-zone
+	 * @return zoned date-time
 	 */
 	@Nullable
-	String zonedDateTime();
+	ZonedDateTime zonedDateTime();
```

Comment:
We need to keep the backwards compatibility.

Reply:
@violetagg do you mind if I add deprecation notice to current `String zonedDateTime();` in favor of new ZonedDateTime-typed field then?

Reply:
Yes please do so

```
@deprecated as of 1.0.6. Prefer using ....This method will be removed in version 1.2.0.
```

Thanks